httphandler/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 		ctx = logger.InitOtel("kubescape",
 			os.Getenv(cautils.BuildNumber),
 			os.Getenv("ACCOUNT_ID"),
+			os.Getenv("CLUSTER_NAME"),
```

Comment:
@dwertent  I think the correct variable should be `KS_CONTEXT`
https://github.com/kubescape/helm-charts/blob/e4dc0efd5f9a7fa097486975193b8f5197f761bd/charts/kubescape-cloud-operator/templates/kubescape/deployment.yaml#L83

Reply:
The `CLUSTER_NAME` is the identifier, not the context...